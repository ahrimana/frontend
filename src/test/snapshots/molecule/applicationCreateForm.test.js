/* eslint no-undef: 0 */
/* eslint no-unused-expressions: 0 */
/* eslint no-sequences: 0 */
/* eslint quotes: 0 */

import React from 'react';
import CreateApplicationForm from '../../../components/molecules/application/createForm';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import getMuiTheme from 'material-ui/styles/getMuiTheme';

describe("Create application form(snapshot)", () => {
  const setup = () => {
    const props = {
      serviceId: "808080",
    };

    const muiTheme = getMuiTheme();
    const context = {
      context: { muiTheme },
      childContextTypes: { muiTheme: React.PropTypes.object },
    };
    return shallow(<CreateApplicationForm { ...props } />, { ...context });
  };
  it("Create application form render form", () => {
    const component = setup();
    expect(toJson(component)).toMatchSnapshot();
  });
});
