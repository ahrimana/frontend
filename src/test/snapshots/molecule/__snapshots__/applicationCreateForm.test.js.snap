exports[`Create application form(snapshot) Create application form render form 1`] = `
<form
  onSubmit={[Function]}>
  <div>
    <TextField
      disabled={false}
      floatingLabelFixed={false}
      floatingLabelText="Name"
      fullWidth={false}
      multiLine={false}
      onChange={[Function]}
      required={true}
      rows={1}
      type="text"
      underlineShow={true} />
  </div>
  <div>
    <TextField
      disabled={false}
      floatingLabelFixed={false}
      floatingLabelText="Galaxy Role"
      fullWidth={false}
      multiLine={false}
      onChange={[Function]}
      required={true}
      rows={1}
      type="text"
      underlineShow={true} />
  </div>
  <div>
    <RaisedButton
      disabled={false}
      fullWidth={false}
      label="Create"
      labelPosition="after"
      primary={false}
      secondary={false}
      type="submit" />
  </div>
</form>
`;
